src/tests/modify.test.js

```diff
@@ -397,3 +397,89 @@ describe('modify() - basic mutations', () => {
     });
   });
 });
+
+describe('modify() - reoder fields', () => {
```

Comment:
I've rebased this (Order PR) with main https://github.com/remoteoss/json-schema-form/pull/83, including the new specs discussed in the previous threads. Ready for a last review :)